odoo/modules/registry.py

```diff
@@ -213,9 +213,15 @@ def get_manual_fields(self, cr, model_name):
         if self._fields_by_model is None:
             # Query manual fields for all models at once
             self._fields_by_model = dic = defaultdict(dict)
+            id2name = defaultdict(dict)
```

Comment:
Using `defaultdict` here is confusing, because the loop below maps some `int` to `str`...
I would simply declare it as `id2name = {}`.